auditwheel/wheel_abi.py

```diff
@@ -97,8 +100,12 @@ def analyze_wheel_abi(wheel_fn: str):
         for p in load_policies()
     }
 
-    elftree_by_fn, external_refs_by_fn, versioned_symbols, has_ucs2, uses_PyFPE_jbuf= \
-            get_wheel_elfdata(wheel_fn)
+    wheel_elfdata = get_wheel_elfdata(wheel_fn)
+    elftree_by_fn = wheel_elfdata[0]
+    external_refs_by_fn = wheel_elfdata[1]
+    versioned_symbols = wheel_elfdata[2]
+    has_ucs2 = wheel_elfdata[3]
+    uses_PyFPE_jbuf = wheel_elfdata[4]
```

Comment:
IMO we should really just change this function to return a dict instead of a tuple, describing what the return arguments are. Anything more than 2 or 3 posargs is difficult to make sense of and these new assignments sort of lay that bare.